lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h

```diff
@@ -130,6 +132,9 @@ XX(down, string, none, down, __VA_ARGS__)
 XX(identifier, string, none, id, __VA_ARGS__) \
 XX(command, TunnelCommand, none, command, __VA_ARGS__)
 
+#define TNL_DELETE_IDENTITY(XX, ...) \
+XX(identifier, string, none, identifier, __VA_ARGS__)
```

Comment:
I see, all of these `tnl_*_identlty` models have an `identifier` field. Some of them existed before this PR.

Why does this one use `identifier` for the C field name when the others use `id`? They should at least be consistent for all of the related models.

Reply:
I changed the id field to identifier in all the models